src/main/java/edu/utdallas/davisbase/host/Host.java

```diff
@@ -39,9 +42,68 @@ public Host(HostConfiguration configuration, BufferedReader reader, PrintWriter
     this.writer = writer;
   }
 
-  public String readStatement() throws IOException {
-    // TODO Implement Host.readStatement()
-    throw new NotImplementedException();
+  static String prompt = "davisql> ";
+	static String copyright = "Team Blue";
+  static String version = "v1.0";
+  static boolean exit_prompt = false;
+
+	static Scanner scanner = new Scanner(System.in).useDelimiter(";");
+
+  public String readStatement(String input) throws IOException {
+
+    PromptScreen();
+
+		String userInput = "";
+
+		System.out.print(prompt);
+    userInput = scanner.next().replace("\n", " ").replace("\r", "").trim().toLowerCase();
+    return userInput;
+  }
+
+
+  public static void PromptScreen() {
+		System.out.println(line_chars("-",80));
+    System.out.println("Welcome to DavisBase");
+		System.out.println("DavisBase Version " + version);
+		System.out.println(copyright);
+		System.out.println("\nType \"help;\" to display supported commands.");
+		System.out.println(line_chars("-",80));
+	}
+
+
+	public static String line_chars(String s,int num) {
+		String a = "";
```

Comment:
Use `StringBuilder`. There's absolutely no reason to use `String` here. It's horrendously inefficient.